components/VotesList/VotesListItem.tsx

```diff
@@ -380,13 +383,20 @@ export function VotesListItem({
               onSelect={onSelectVote}
             />
           ) : (
-            <TextInput
-              value={selectedVote ?? getDecryptedVoteAsString() ?? ""}
-              onInput={selectVote}
-              maxDecimals={maxDecimals}
-              type="number"
-              disabled={!signer}
```

Comment:
I think disabling the text input when there's no signer is from old code as well. We don't disable the dropdown, we just disable the commit button.